src/Framework/Framework.slnf

```diff
@@ -19,7 +19,8 @@
       "src\\Extensions\\Features\\src\\Microsoft.Extensions.Features.csproj",
       "src\\FileProviders\\Embedded\\src\\Microsoft.Extensions.FileProviders.Embedded.csproj",
       "src\\FileProviders\\Manifest.MSBuildTask\\src\\Microsoft.Extensions.FileProviders.Embedded.Manifest.Task.csproj",
-      "src\\Framework\\Analyzer\\src\\Microsoft.AspNetCore.App.Analyzers.csproj",
+      "src\\Framework\\AspNetCoreAnalyzers\\src\\Analyzers\\Microsoft.AspNetCore.App.Analyzers.csproj",
+      "src\\Framework\\AspNetCoreAnalyzers\\src\\CodeFixes\\Microsoft.AspNetCore.App.CodeFixes.csproj",
```

Comment:
Why aren't these in AspNetCore.sln❔ I don't think the solution filter will include these projects as-is.

Reply:
In addition, this file and AspNetCore.sln are missing the renamed src/Framework/AspNetCoreAnalyzers/src/Analyzers/Microsoft.AspNetCore.App.Analyzers.csproj

These problems also exist in release/6.0

Reply:
Ooof! I'll fix this up. I got a code-check warning about removing the old references but not one about adding the new ones.

Is there a chance we can add a code-check validation for that?

Reply:
> Is there a chance we can add a code-check validation for that?

Sure. I'll work on such a thing later today.

Reply:
Cool! Thanks. I'll submit a PR fixing this up in release/6.0

Reply:
src/Framework/AspNetCoreAnalyzers/src/Analyzers/Microsoft.AspNetCore.App.Analyzers.csproj is still not mentioned here nor in AspNetCore.sln

Reply:
It's added in https://github.com/dotnet/aspnetcore/pull/36379/files#diff-d9f61b2475cc9938175a9c3e49e42bc4109ac983a29c73560932b2dc28fb4157R22 and https://github.com/dotnet/aspnetcore/pull/36379/files#diff-cd977e0a76b37d35c04d9d819ea66ef8a35d9ef7f86a9a7c774d751e8119db4fR1653.

Reply:
I keep copying the wrong project path ☹️ I meant src/Framework/AspNetCoreAnalyzers/**test**/Microsoft.AspNetCore.App.Analyzers.**Test**.csproj